.github/workflows/composer.test.yml

```diff
@@ -24,6 +24,7 @@ jobs:
       with:
         php_version: 8.2
         command: install
+        args: --ignore-platform-req=ext-pcntl
```

Comment:
I do lack the PHP knowledge to be able to tell whether this is ok to do. Things continued to work for me locally :tm:, but maybe that's a happy coincidence?